paimon-core/src/test/java/org/apache/paimon/catalog/FileSystemCatalogTest.java

```diff
@@ -44,4 +45,11 @@ public void testListDatabasesWhenNoDatabases() {
         List<String> databases = catalog.listDatabases();
         assertThat(databases).isEmpty();
     }
+
+    @Test
+    public void testRenameDatabase() throws Exception {
+        catalog.createDatabase("db1", false);
+        catalog.renameDatabase("db1", "db2", false);
+        assertThat(catalog.listDatabases()).isEqualTo(Lists.newArrayList("db2"));
```

Comment:
Could we add the test case to rename to the same name of other database?

Reply:
I will do it.